Codegen/src/GeneratorHelper.cs

```diff
@@ -100,4 +100,4 @@ public static string ComputeIndexHashName(string qualified_name, Node sourceNode
         }
 
     }
-}
+}
```

Comment:
No need to change the file.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1648/commits/c148777c31d93bcbf91b96ae92881f3f876a21d9